pkg/solana/txm/txm.go

```diff
@@ -170,9 +170,10 @@ func (txm *Txm) sendWithRetry(ctx context.Context, baseTx solanaGo.Transaction,
 
 	// base compute unit price should only be calculated once
 	// prevent underlying base changing when bumping (could occur with RPC based estimation)
+	baseFee := txm.fee.BaseComputeUnitPrice()
 	getFee := func(count int) fees.ComputeUnitPrice {
 		fee := fees.CalculateFee(
-			txcfg.BaseComputeUnitPrice,
+			baseFee,
```

Comment:
Eyes 👀. Added this change here so we avoid merging 2 PRs: https://github.com/smartcontractkit/chainlink-solana/pull/892

Reply:
Sent a message async with a better explanation but don't think we need this change

Reply:
reverted